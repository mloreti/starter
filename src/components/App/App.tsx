import * as React from 'react';
import { Provider } from 'react-redux';

import store from '../../store';

export default () => (
  <Provider store={store}>
    <h1>Hello world</h1>
  </Provider>
)
