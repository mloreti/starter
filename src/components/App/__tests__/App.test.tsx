import { ShallowWrapper, shallow } from 'enzyme';
import * as React from 'react';

import App from '../App';

describe('App', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(<App />);
  });

  it('renders properly', () => {
    expect(component).toMatchSnapshot();
  });
});