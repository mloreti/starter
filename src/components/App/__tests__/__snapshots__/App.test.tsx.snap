// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders properly 1`] = `
<h1>
  Hello world
</h1>
`;
